nbs/60_medical.imaging.ipynb

```diff
@@ -80,6 +80,56 @@
     "_all_ = ['DcmDataset', 'DcmTag', 'DcmMultiValue', 'dcmread', 'get_dicom_files']"
```

Comment:
<p>This docstring is less accurate than what was there before - suggest you revert it (since "the pixel_array" is confusing, since it's a computed property)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>reverted back to the previous docstring</p>